index.html

```diff
@@ -96,6 +96,11 @@ <h3 class="panel-title">
                         </div>
                         <div class="list-group scrollable-list">
                              <a class="list-group-item">
+                                <h4 class="list-group-item-heading">Friday, August 5, 2016</h4>
+                                <p class="list-group-item-text">The DDF Development Team has updated the recommended system configuration for DDF. Please ensure a Java version of  or later to take advantage of performance enhancements.

```

Comment:
Maybe this is a silly question, but how does this display the version? I'm seeing "    ".